commands/credentials/url.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const co = require('co')
+const cli = require('heroku-cli-util')
+const url = require('url')
+
+function * run (context, heroku) {
+  const host = require('../../lib/host')
+  const fetcher = require('../../lib/fetcher')(heroku)
+  const util = require('../../lib/util')
+
+  const {app, args, flags} = context
+
+  let db = yield fetcher.addon(app, args.database)
+  let cred = flags.name || 'default'
+  if (util.starterPlan(db) && cred !== 'default') throw new Error('This operation is not supported by Hobby tier databases.')
```

Comment:
This error message is slightly misleading: the action is supported; just not with that option.